tests/acceptance/AdvancedSettingsAnonDataToggleCest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class AdvancedSettingsAnonDataToggleCest {
+  function toggleAnonDataSetting(\AcceptanceTester $I) {
+    $I->wantTo('Confirm anon data settings can be toggled');
```

Comment:
Could we use the full word? I had no idea what `anon` is until I opened full advanced.html. Also the class could be just `AdvancedSettingsCest` maybe? To possibly contain multiple test in future.

Reply:
Sure thing. I'll probably create new classes for the other advanced settings tests, so I changed the names to include anonymous instead of anon.